SPECS/perl-generators/perl-generators.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "generators-1.11.tar.gz": "1288728c9d30cf9319151b34537d5f6ad1deb27a736dd956748f43b4da717f13"
```

Comment:
Ask: could we re-name this binary to `perl-generators-%{version}.tar.gz`? This name is a bit generic and we run into potential name conflict issues (although not very likely). Not a big deal, though.

Reply:
We can name the tarballs whatever we want and still keep a valid URL. A simple way to do that is this (notice the `#` after the download URL):
```
Source0:        [working_download_url]#/%{name}-%{version}.tar.gz
```

We do this in `hyperscan.spec` for instance.